packages/react-ui-ag/src/Listings/MobileMapListing.js

```diff
@@ -99,6 +108,11 @@ export default class MobileMapListing extends Component {
     }
   }
 
+  @autobind
+  handlePhotoCarouselSlide(index) { // eslint-disable-line no-unused-vars
```

Comment:
I'm guessing there is a reason, but I'm confused why you're giving this function a parameter at all